src/utils.rs

```diff
@@ -1022,30 +1068,28 @@ fn parse_punctuated_nested_meta(
                     ));
                 }
 
-                // `not(ignore)` does not make much sense
-                if path.is_ident("ignore") && is_top_level {
-                    info.enabled = Some(false);
-                } else if path.is_ident("forward") {
-                    info.forward = Some(is_top_level);
-                } else if path.is_ident("owned") {
-                    info.owned = Some(is_top_level);
-                } else if path.is_ident("ref") {
-                    info.ref_ = Some(is_top_level);
-                } else if path.is_ident("ref_mut") {
-                    info.ref_mut = Some(is_top_level);
-                } else if path.is_ident("source") {
-                    info.source = Some(is_top_level);
-                } else if path.is_ident("backtrace") {
-                    info.backtrace = Some(is_top_level);
-                } else {
-                    return Err(Error::new(
-                        path.span(),
-                        format!(
-                            "Attribute doesn't support parameter `{}` here",
-                            quote! { #path }
-                        ),
-                    ));
-                };
+                let attr_name = path.get_ident().unwrap().to_string();
+                match (wrapper_name, attr_name.as_str()) {
```

Comment:
This match is not equivalent to the if statement before. Some things will now error that didn't before, e.g. `not(ref)`. Although doesn't make much sense, but now it errors where it did not before. 

I guess this is on purpose, because this could now get confusing with something like `types(i32, i64), not(ref(types(i64)))`.

Reply:
@JelteF yup, this is on purpose, because now we have more nesting layers for attribute arguments. So stricter validation here only helps.